docs/getstarted/whatis.md

```diff
@@ -20,4 +20,6 @@ Tools included:
 
 ![Annotate Screenshot](/img/annotate/annotate.png)
 
-Audit enables project leaders to track every change or annotation to an image for monitoring and presenting to regulators as necessary
+[Audit](interfaces/audit) enables project leaders to track every change or annotation to an image for monitoring and presenting to regulators as necessary.
```

Comment:
Should things like "Audit" be "AUDIT"? I know I capitalise stuff but do we have a brand guideline on this? @RoseBunker @joshuajames-smith ?

Reply:
Brand guidelines are _pending_ but all products should be capitalised, i.e. ANNOTATE, CURATE, AUDIT, MANAGE.